stl/inc/xutility

```diff
@@ -537,6 +537,20 @@ struct less_equal<void> {
     using is_transparent = int;
 };
 
+template <class _Ty, class = void>
+_INLINE_VAR constexpr bool _Is_transparent_v = false;
+
+template <class _Ty>
+_INLINE_VAR constexpr bool _Is_transparent_v<_Ty, void_t<typename _Ty::is_transparent>> = true;
+
+template <class _Ty>
+struct _Is_transparent : bool_constant<_Is_transparent_v<_Ty>> {};
+
+#ifdef __cpp_lib_concepts // TRANSITION, GH-395
+template <class _Ty>
+concept _Transparent = _Is_transparent_v<_Ty>;
+#endif // __cpp_lib_concepts
+

```

Comment:
It was decided in #3736 that these stuffs should be in `<xmemory>`. So perhaps they should be dropped from `<xutility>`.